webapp/channels/src/components/admin_console/custom_url_schemes_setting.tsx

```diff
@@ -63,11 +51,11 @@ export default class CustomURLSchemesSetting extends
     };
 
     render() {
-        const label = Utils.localizeMessage('admin.customization.customUrlSchemes', 'Custom URL Schemes:');
-        const helpText = Utils.localizeMessage(
-            'admin.customization.customUrlSchemesDesc',
-            'Allows message text to link if it begins with any of the comma-separated URL schemes listed. By default, the following schemes will create links: "http", "https", "ftp", "tel", and "mailto".',
-        );
+        const label = this.props.intl.formatMessage({id: 'admin.customization.customUrlSchemes', defaultMessage: 'Custom URL Schemes:'});
+        const helpText = this.props.intl.formatMessage({
+            id: 'admin.customization.customUrlSchemesDesc',
+            defaultMessage: 'Allows message text to link if it begins with any of the comma-separated URL schemes listed. By default, the following schemes will create links: "http", "https", "ftp", "tel", and "mailto".'
+        });
```

Comment:
instead of defining another variable can you please declare them inline how you have done for Input's placeholder

Reply:
yes sounds good, make sure you run `npm run fix` on the webapp folder so auto-style anything missed